### PR TITLE
explain how to set app language

### PR DESCRIPTION
**Pull request by AlexP11223**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,15 @@ Basic steps to translate using Qt Linguist:
...
 
+## How to change app language
+
+By default Flameshot chooses the translation language based on the OS settings.
+Currently there is no option in config to override this.
+
+On Linux you can set the `LANGUAGE` environment variable.
+For example, by launching via terminal: `LANGUAGE=ru_RU:ru flameshot`.
+Or if you are a developer and building/running Flameshot via Qt Creator, you can do this via [project settings](https://i.imgur.com/1LYkHyh.png).
+
```
Comment (ZetaoYang) [inline README.md:32]: I suggest uploading the screenshots to github instead of imgur.

**Comment by mmahmoudian** [comment]
@AlexP11223 I think this PR is in a wrong repo. As far as I understand, this repo (translation-instruction) is only to clarify how to start translation. What you explained here is related to "how ro use the software", therefore it should be added to the [main repo](https://github.com/flameshot-org/flameshot) and also to [the website](https://github.com/flameshot-org/flameshot-org.github.io).

**Comment by AlexP11223** [comment]
It should be mainly useful for translators who want to check different languages or translate and test language that is not their OS language. Other users usually should be ok with the default behavior, it doesn't make much sense to change language just for one app (unless it's determined incorrectly, e.g. by Windows locale for non-unicode apps instead of the preferred UI language settings, but then it's a bug).

**Comment by mmahmoudian** [comment]
@AlexP11223 Thanks for explanation. Now it makes sense to me. 👍

**Review by mmahmoudian — APPROVED** [review]

**Review by ZetaoYang — APPROVED** [review]

**Pull request merged**